Source/WebKit/UIProcess/API/Cocoa/WKDownload.mm

```diff
@@ -148,20 +150,20 @@ void decideDestinationWithSuggestedFilename(WebKit::DownloadProxy& download, con
     }
 
 #if HAVE(MODERN_DOWNLOADPROGRESS)
-    void decidePlaceholderPolicy(WebKit::DownloadProxy& download, CompletionHandler<void(WebKit::UseDownloadPlaceholder)>&& completionHandler)
+    void decidePlaceholderPolicy(WebKit::DownloadProxy& download, CompletionHandler<void(WebKit::UseDownloadPlaceholder, const WTF::URL&)>&& completionHandler)
     {
         if (!m_respondsToDecidePlaceholderPolicy) {
-            completionHandler(WebKit::UseDownloadPlaceholder::No);
+            completionHandler(WebKit::UseDownloadPlaceholder::No, WTF::URL());
```

Comment:
You can probably just use { } for places where we are just sending an empty thing.